assets/question_sets/strings.js

```diff
@@ -26,5 +26,5 @@ globalData.questionSets['strings'] = {  // eslint-disable-line dot-notation
       "I will provide you with feedback."
     ].join('')
   ],
-  questionIds: ['reverseWords', 'parens', 'i18n', 'rle']
+  questionIds: ['reverseWords', 'parens', 'i18n', 'rle', 'bomberman']
```

Comment:
Thanks! The problems have been fixed, I'm now working on CLA issue. This branch may be deleted in order to pass the CLA check.